build.py

```diff
@@ -1185,20 +1188,28 @@ def windeployqt_list_source(
         target: pathlib.Path,
         windeployqt: pathlib.Path,
 ) -> typing.Iterable[pathlib.Path]:
-    try:
-        process = report_and_check_call(
-            command=[
-                windeployqt,
-                '--dry-run',
-                '--list', 'source',
-                target,
-            ],
-            stdout=subprocess.PIPE,
-            # ugh, 3.5
-            # encoding='utf-8',
-        )
-    except subprocess.CalledProcessError as e:
-        raise DependencyCollectionError(target) from e
+    process = report_and_check_call(
+        command=[
+            windeployqt,
+            '--dry-run',
+            '--list', 'source',
+            target,
+        ],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        # ugh, 3.5
+        # encoding='utf-8',
+        check=False,
+    )
+
+    if process.returncode:
+        if (process.returncode > 0 and
+            'does not seem to be a Qt executable'
+            in process.stderr.decode('utf-8')):
+            # This can happen with some utilities like
+            # `rcc.exe`, which we still want to include.
+            return []
+        raise DependencyCollectionError(target, process)
```

Comment:
It looks like we can just pull the return code and stderr off the exception.

```suggestion
    try:
        process = report_and_check_call(
            command=[
                windeployqt,
                '--dry-run',
                '--list', 'source',
                target,
            ],
            stdout=subprocess.PIPE,
            stderr=subprocess.PIPE,
            # ugh, 3.5
            # encoding='utf-8',
        )
    except subprocess.CalledProcessError as e:
        if 'does not seem to be a Qt executable' in e.stderr.decode('utf-8'):
            # This can happen with some utilities like
            # `rcc.exe`, which we still want to include.
            return []
 
        raise DependencyCollectionError(target) from e
```

Reply:
Yes, much better.